cfgBuilder/README.md

```diff
@@ -42,5 +42,5 @@ An input JSON config looks like this:
   ]
 }
 ```
-
-Running `go run main.go <json-path> <output-path>` will produce config files for ChainBridge, and (optional) place them in the specified path. For each relayer in `relayers` a corresponding config will be generated.
\ No newline at end of file
+To run `cfgBuilder`, first run `make install-cfgBuilder` and build `make build-cfgBuilder` 
+Running `./build cfgBuilder <json-path> <output-path>` will produce config files for ChainBridge, and (optional) place them in the specified path. For each relayer in `relayers` a corresponding config will be generated.
```

Comment:
install and build are equivalent, install just has an extra step that makes the command available in your terminal.